.github/workflows/awsfulltest.yml

```diff
@@ -26,5 +26,5 @@ jobs:
             {
               "outdir": "s3://${{ secrets.AWS_S3_BUCKET }}/chipseq/results-${{ github.sha }}"
             }
-          profiles: '[ "test_full", "aws_tower" ]'
-
+          profiles: test_full,aws_tower
+          pre_run_script: 'export NXF_VER=21.10.3'
```

Comment:
```suggestion
```

Reply:
Should be able to remove this now because the latest version on Tower has been bumped.